src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -1159,7 +1159,7 @@ private void EmitArray<T>(Type arrType, IEnumerable<T> items, Action<T, int> ite
                     //can't handle these with compile time indexing.
                     //TODO remove IList from this if stmt when we figure out function call return wrapping behavior.
                     //this is still a problem for BuiltIn Dictionaries that are wrapped in an IList.
-                    if (arrayT == null || arrayT == typeof(IList) || arrayT == typeof(object))
+                    if (arrayT == null || arrayT == typeof(IList) || arrayT == typeof(object) || arrayT == typeof(DSASM.CLRStackValue))
```

Comment:
so this means we will not support indexing into the result of a replicated call right ?

Reply:
It means that we for now support it by calling the `valueAtIndex` method.